ocaml/default.nix

```diff
@@ -534,6 +518,10 @@ with oself;
   graphql-async = callPackage ./graphql/async.nix { };
 
   graphql_ppx = callPackage ./graphql_ppx { };
+  graphql-cohttp = osuper.graphql-cohttp.overrideAttrs (o: {
+    # https://github.com/NixOS/nixpkgs/pull/170664
+    nativeBuildInputs = [ ocaml dune findlib crunch ];
```

Comment:
`o.nativeBuildInputs ++ [ crunch ]`

Reply:
`nativeBuildInputs` brings in `ocaml-crunch` which depends on `pkgs.ocamlPackages`. the version there might be different. check https://github.com/NixOS/nixpkgs/pull/170664

Reply:
Ah 👌